src/main/webapp/itempagescript.js

```diff
@@ -38,32 +42,35 @@ async function createSelectedItemCard(entertainmentItem) {
  * Sends comment data and ItemId to Servlet
  */
 async function sendFormData() {
+  const comment = $('#comment').val();
   const itemId = getUrlParam('itemId');
-  const comment = document.getElementById('comment');
-  fetch(
-    `/itempagedata?=${itemId}`,
-    { method: 'post', body: JSON.stringify(comment) })
-}
+  $.post('/itempagedata', { comment: comment, itemId: itemId })
+    .done(function() {
+      window.location.reload();
+    })
+    .fail(function() {
+      console.log('Failed to send form data');
+    });
 
 /**
  * Function which builds the comment element from the ItemPageData object
  */
 function getItemPageComments(comments) {
-  const commentContainer = document.getElementById('comment-container');
+  const commentContainer = $('#comment-container')
   comments.forEach((commentDataManager) => {
     const date = new Date(commentDataManager.timestampMillis);
-    commentContainer.appendChild(createListElement(
-      commentDataManager.message + ' - ' +
-      '(' + date.toUTCString() + ')'));
+    commentContainer.append(createListElement(
+      commentDataManager.comment + ' - ' +
+      '(' + date.toLocaleString() + ')'));
   });
 }
 
 /**
- * Creates list element which houses comments
+ * Creates list element which houses comments.
  */
 function createListElement(comment) {
-  const liElement = document.createElement('li');
-  liElement.innerText = comment;
+  var liElement = document.createElement('li');
```

Comment:
Use ```const``` instead of ```var``` since the object is not going to get re-assigned. Make sure to check Google's Javascript style guide: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let